TGM/src/main/java/network/warzone/tgm/modules/MatchResultModule.java

```diff
@@ -43,7 +43,11 @@ public void onMatchResult(MatchResultEvent event) {
             if (spectators.containsPlayer(player)) {
                 player.playSound(location, Sound.ENTITY_WITHER_DEATH, 1000, 1);
             } else {
-                player.setGameMode(GameMode.ADVENTURE);
+                if (TGM.get().getConfig().getConfigurationSection("map").getBoolean("post-block-break") && player.hasPermission("tgm.post.break")) {
```

Comment:
`TGM.get().getConfig().getConfigurationSection("map").getBoolean("post-block-break")`
Can be simplified and provide a default value with
`TGM.get().getConfig().getBoolean("map.post-block-break", false)`

Reply:
only going off what i saw in other tgm code